src/app/country/country-list/country-list.component.html

```diff
@@ -0,0 +1,4 @@
+<span class="title">Country List</span>
+<div class="list-item" *ngFor="let item of countryList$ | async">
```

Comment:
What's the benefit of using ``async`` instead of ``subscribe()`` in the component?

Reply:
Because you don't need to unsubscribe the observable ones that the component is destroyed.